perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/PropagationMaintainer.java

```diff
@@ -186,11 +191,11 @@ private void rescheduleErrorTasks() {
 				howManyMinutesAgo = task.getDelay() + 1;
 			}
 
-			log.trace("[{}] Task in ERROR state completed {} minutes ago: {}.", new Object[]{task.getId(), howManyMinutesAgo, task});
+			log.trace("[{}] Task in ERROR state completed {} minutes ago: {}.", task.getId(), howManyMinutesAgo, task);
 
 			// If DELAY time has passed, we reschedule...
 			int recurrence = task.getRecurrence() + 1;
-			LocalDateTime twoDaysAgo = LocalDateTime.now().minusDays(2);
+			LocalDateTime toManyHoursAgo = LocalDateTime.now().minusHours(oldRescheduleHours);
```

Comment:
Probably just a typo here: "toManyHoursAgo" -> "tooManyHoursAgo".

Reply:
Thanks for noticing, fixed.